### DISCUSSION TITLE
Request for new feature: config para remover páginas cinzentas do template da Uminho

### DISCUSSION THREAD

**Discussion by vascoalramos** [joaomlourenco/novathesis#257] [Q&A] [answered]
Para a versão digital as páginas cinza que são incluídas entre as páginas da capa não são necessárias. Existe alguma forma de as remover, ou caso não haja, seria possível criar uma configuração para tal?

Obrigado!

**Comment by joaomlourenco** [comment]
@b-pereira usually take scare of uminho stuff. I'll talk with home about this. But we can implement the requested option. 🙂

**Comment by b-pereira** [comment]
While the maintainer is more than able to do the request, I only have some notes to add (this is more a rant than anything). 

So @vascoalramos, the last 2 weeks were a bit strange. With one colleague ***doctoral thesis***, the School of Engineering's Scientific Council **mandated explicitly** that the cover, **back-cover** and spine **must** be included in the file. Following the _Order RT-31/2019_'s rules _verbatim_, the grey back cover and spine should be included in the dissertation, regardless of being digital or printed. However, **many** dissertations were delivered **with only the front-cover** and nobody said anything. Also, the Informatics Engineering's Course Board stated, informally, that there's a differentiation between digital and printed, but that isn't in the _Order RT-31/2019_. Also note that the Course Board has nothing to do with Scientific Council, and both can say different things. However, the Scientific Council is above the Course Board and dictate the rules.

> Para a versão digital as páginas cinza que são incluídas entre as páginas da capa não são necessárias.[^1]

I don't know how true your quote's assertion is anymore, because I just don't know how the bureaucrats will respond. To follow the rules *strictly* or *loosely*? I have seen **dissertations** from other schools with the ***full-cover with spine*** and ***back-cover*** and they were not to be printed. Also, as I said, **many** dissertations were delivered ***with only the front-cover***. But the official template from Informatics Engineering Master's Website doesn't also have the grey cover or the spine.

So, if **many** dissertations were delivered ***with only the front-cover*** and nobody said anything, therefore do as you please. But again is like the rules are seldomly applied and that leaves some frustration. To end on a positive note, check the following quotes[^2], that illustrate the situation 😁:

> "To be or not to be"
> — William Shakespeare

>"To be is to do." 
>      — Aristotle

>"To do is to be"
> — J. P. Sartre

> "Do be do be do"
> — F. Sinatra


> "What is to be done?"
> — Lenin

> Do It!
> — J. Rubin

> O.K.! O.K.!
> — T. Mann.

🤣🤣🤣🤣🤣🤣 

[^1]: **Please write in English in this forum**.
[^2]: “To Be Is To Do” “To Do Is To Be” “Do Be Do Be Do” – Quote Investigator. (2021, November 27). Retrieved from https://quoteinvestigator.com/2013/09/16/do-be-do

**Reply by vascoalramos** [reply]
Sorry I didn't write the original question in English :sweat_smile: 

About the question itself... Yes, as far as my knowledge goes many master dissertations have been delivered without the grey pages, especially because de "official" templated doesn't have them and the current head of the Informatics Department (he is the adviser of some colleagues of mine) said it was fine

**Reply by b-pereira** [reply]
@vascoalramos the head of the Informatics Department, you mean _prh_? _als_ told me that, like the rest of the Course Board as well _jja_ (_jja_ was in charge of the new template, but the final one was made by _jno_). I talked to them in person (less _jno_). But again, the School of Engineering rejected one doctoral thesis, just like it was with the main one. But to be honest, I remember seeing a previous thesis, from 2013 (I think) which the adviser was _prh_ and it had the full cover and back cover. It's easier to do so with `pdfpages`.

**Reply by vascoalramos** [reply]
Okay... It is very ambiguous then... Even so, could give me some directions on how to do it (at least for the master pre dissertation report - rpd)?

Thanks!

**Reply by joaomlourenco** [reply]
I don't mind adding this option if @b-pereira gives me a "green light".  😉
Do you @b-pereira?

**Reply by b-pereira** [reply]
There is the issue  [#256] that I created because of a bug in the pre-dissertation reports, as well thesis proposal and thesis plan. The options `mscplan`, `phdprop` and `phdplan` weren't implemented @joaomlourenco. Also, if I give a 🚦 🟢?? You're in charge!! @vascoalramos a quick fix with this template, I don't know how to do it. With my variant, ya! But you don't need to. Also, the PDR (RPD in _pt_) is in place if is just to deliver the report. And it's the PDR - I delivered mine with the `ArsClassica` article template and with the [PDF Form](https://mei.di.uminho.pt/sites/default/files/Capa-MIMEI.pdf) on MIEI/MEI page. Nobody cares at this stage with the formatting. They only care about **the content** and **presentation** - at least for me and some others. At the time (this February), when I delivered it, it was just a bureaucratic formality to see what you did and give you hints. I don't know if it changed back to 2013s, where it was somewhat formal - like to the adviser sign a paper, and so on. 

In short, if you did the work and have something that you can call a presentation, you'll be fine. Even if the presentation and work are not up to standard, well, in that case they call .. ahum.. your attention. But they don't give you a grade. In 2013s, yes they did.

**Reply by vascoalramos** [reply]
I see, nice to know! Maybe give this possibilty to the `mscplan` and `phdplan`?

**Comment by joaomlourenco** [comment, marked as answer]
@vascoalramos and @b-pereira, is this problem still open?  Can you please state clearly what you want?  And if you want a hack or something definitive? If the latter, permanent or as an option in the template?

Thanks.

**Reply by b-pereira** [reply]
@joaomlourenco he added a custom cover.

**Reply by vascoalramos** [reply]
Yes, exactly, no need to add the option.

**Reply by joaomlourenco** [reply]
@b-pereira @vascoalramos 
I am lost…
**Would such option make sense?**  
If it does, then it is not because @vascoalramos solved his problem that the issue is solved.
So, please, clarify:
1) Does it make sense to have an option for removing the colored back-page from cover and front-page from back cover?
2) If yes, should this be the default behavior for document types "mscpan", "phdplan", and "phdprop"?
Thank you.

**Reply by vascoalramos** [reply]
1. Yes, I think it does.
2. Yes, it should be the default behaviour for those document types.

**Reply by b-pereira** [reply]
I second that. I would go so far as to say that the default behaviour for `msc` would be no back-cover and no cover and back cover verses with the option to add them if intended. But the options that @vascoalramos agreed with would suffice. There's no need to complicate things further.

**Comment by joaomlourenco** [comment]
@b-pereira e @vascoalramos, esta feature já está disponível na v6.9.1.
